2025-06-13 12:52:40,225 [INFO] Iniciando aplicación OBD2 Scanner...
2025-06-13 12:52:41,344 [DEBUG] Using selector: _Selector
2025-06-13 12:52:41,809 [CRITICAL] Excepción no capturada:
Traceback (most recent call last):
  File "/Users/ignacioeltit/git/proyecto01junio/ob2_nuevo7junio/scanner-obd2/src/main.py", line 507, in <module>
    win = DataVisualizer(lambda: {}, pid_manager=None, elm327=None, parse_pid_response=None, backend=backend)
  File "/Users/ignacioeltit/git/proyecto01junio/ob2_nuevo7junio/scanner-obd2/src/ui/data_visualizer.py", line 121, in __init__
    self.init_ui()  # <-- Primero inicializa la UI (crea self.tabs)
  File "/Users/ignacioeltit/git/proyecto01junio/ob2_nuevo7junio/scanner-obd2/src/ui/data_visualizer.py", line 251, in init_ui
    self.graphs_layout.addWidget(self.graph_plot)  # PyQtGraph PlotWidget ya es un QWidget compatible
TypeError: 'PySide6.QtWidgets.QBoxLayout.addWidget' called with wrong argument types:
  PySide6.QtWidgets.QBoxLayout.addWidget(PlotWidget)
Supported signatures:
  PySide6.QtWidgets.QBoxLayout.addWidget(arg__1: PySide6.QtWidgets.QWidget, /, stretch: Optional[int] = None, alignment: PySide6.QtCore.Qt.AlignmentFlag = Default(Qt.Alignment))
